TimeOnOff buzzer
TimeOnOff md_power
TimeOnOff md_estop
TimeOnOff led_start
TimeOnOff led_stop
TimeOnOff bat_relay
TimeOnOff out_spare1
TimeOnOff out_spare2
---
bool success